sql/core/src/main/scala/org/apache/spark/sql/execution/ReplaceHashWithSortAgg.scala

```diff
@@ -75,6 +75,7 @@ object ReplaceHashWithSortAgg extends Rule[SparkPlan] {
                 partialAgg.child.outputOrdering, sortAgg.requiredChildOrdering.head)) {
               sortAgg.copy(
                 aggregateExpressions = sortAgg.aggregateExpressions.map(_.copy(mode = Complete)),
+                initialInputBufferOffset = 0,
```

Comment:
are you saying that this parameter is not used in complete-mode aggregation, so we should set it to 0?

Reply:
Yes， just directly restore to the default value "0".

cc @dongjoon-hyun

Reply:
Does it cause any issues? If not why do we bother?

We need to carefully review the related source code to make sure this change is safe, and it seems not worthwhile if it's for no good.